src/database/firebase.js

```diff
@@ -109,6 +109,34 @@ export const addTree = async tree => {
   }
 };
 
+/**
+ * same functionality as addTree but for comments:
+ * assigns uuid to comment and adds to commentCollection.
+ */
+export const saveComment = async (comment: Comment) => {
+  try {
+    const ref = await commentCollection.add(comment);
+    commentCollection.doc(ref.id).update({ uuid: ref.id });
+  } catch (e) {
+    console.warn(e);
+    throw e;
+    // TODO: Add error handling.
+  }
+};
+
+export const addComment = async (comment: Comment, uuid: string) => {
+  try {
+    // const ref = await treeCollection.doc(uuid).get();
```

Comment:
We can't merge comments, since that clutters the codebase with dead code. Let's get rid of this. 😄

Reply:
Which one do you want to get rid of?